libshaderc_util/src/spirv_tools_wrapper.cc

```diff
@@ -72,15 +45,21 @@ bool SpirvToolsAssemble(const string_piece assembly, spv_binary* binary,
   *binary = nullptr;
   errors->clear();
 
-  const bool result =
+  const bool success =
```

Comment:
`Assemble()` needs to do a copy to construct a `vector` from `spv_binary`. So for performance reason, I didn't use it here.